src/ocm/components/clusterConfiguration/operators/OdfCheckbox.tsx

```diff
@@ -41,10 +36,10 @@ const OdfHelperText = () => {
 };
 
 const OdfCheckbox = ({ openshiftVersion }: { openshiftVersion?: string }) => {
-  const featureSupportLevelContext = useFeatureSupportLevel();
+  const featureSupportLevelContext = useNewFeatureSupportLevel();
   const fieldId = getFieldId(ODF_FIELD_NAME, 'input');
   const disabledReason = openshiftVersion
```

Comment:
Again, no need to check for `openshiftVersion`. Please remove it from the  `OdfCheckbox` component.